package-lock.json

```diff
@@ -0,0 +1,1123 @@
+{
```

Comment:
please make sure to use yarn.lock instead

Reply:
Should I revert this file in favor of yarn.lock?

Reply:
I would prefer to use yarn.lock. However, if you have a strong opinion why we should use package-lock instead, we can use it and remove yarn.lock :)

Reply:
Nah, not really lol. Fixed in b36f8def44ea7e1f69139510590577888081b668